test/test-functions

```diff
@@ -1736,6 +1736,13 @@ install_basic_tools() {
     image_install -o ldconfig.real
     # for TEST-35-LOGIN
     image_install -o evemu-device evemu-event
+    if command -v expect >/dev/null && command -v tclsh >/dev/null ; then
+        # shellcheck disable=SC2016
+        version="$(tclsh <<< 'puts $tcl_version')"
+
+        image_install expect
+        inst_recursive /usr/lib64/tcl"$version" /usr/share/tcl"$version"
```

Comment:
I missed this, right now the dependencies happen not to be installed in the ubuntu/debian tests, but if they were (intentionally or pulled in accidentally by something else), this would break, as the path is specific to 64bit rpm distros

Reply:
on debian/ubuntu, /usr/bin/tclsh is a symlink to /usr/bin/tclsh<version>

Reply:
there's also both `/usr/lib/tcl<version>` and multi-arch `/usr/lib/x86_64-linux-gnu/tcl<version>`, but no `/usr/share/tcl<version>`

Reply:
Ouch. Will address them.

Reply:
#23930.